src/main/java/io/cdap/plugin/gcp/bigquery/sink/BigQueryOutputFormat.java

```diff
@@ -466,9 +467,9 @@ private String generateQuery(TableReference tableRef) {
         .map(s -> String.format(criteriaTemplate, s, s)).collect(Collectors.joining(", "));
       String orderedBy = orderedByList.isEmpty() ? "" : " ORDER BY " + String.join(", ", orderedByList);
       String sourceTable = String.format(SOURCE_DATA_QUERY, String.join(", ", tableKeyList), orderedBy,
-                                         temporaryTableReference.getProjectId() + "." +
+                                         "`" + temporaryTableReference.getProjectId() + "." +
```

Comment:
similar comment here, a little hard to read. Would be better to make a separate tempTable variable and use String.format() to generate it.